01-filedir.md

```diff
@@ -24,6 +24,12 @@ Several commands are frequently used to create, inspect, rename, and delete file
 To start exploring them,
 let's open a shell window:
 
+> ## Preparation Magic {.callout}
```

Comment:
I have mixed feelings about having them change their prompt.  It does make what the learner sees and the guide shows consistent.  However, we generally try to avoid magic that we don't explain and I'm not sure it is necessary here either.  I propose explaining it similarly to how the filesystem and username differences are now explained, that they should expect this might be different depending on how their computer is set up but that the $ will likely be consistent.